.rubocop.yml

```diff
@@ -72,6 +72,10 @@ Style/WordArray:
 
 ##################### Metrics ##################################
 
+Metrics/BlockLength:
+  Exclude:
+    - 'src/api/config/routes.rb'
```

Comment:
Excluding this file should make the maximum in the todo file smaller, right? if that is the case I'd regenerate the todo file afterwards, so that we have an smaller maximum :wink:

Reply:
I've already done it. It went from 567 to 502.

Reply:
I wouldn't have this preventing the merge.

Reply:
ok